File: baekjoon/dynamic/6359/main.cpp
#include <iostream>
#include <cstdio>
#include <cstring>

using namespace std;

int main()
{
    int testCase;
    scanf("%d", &testCase);
    while(testCase--) {
        int n;
        int ret = 1;
        scanf("%d", &n);
        for(int i=2; i<=n; i++) {
            if(i*i <= n) ret++;
        }
        printf("%d\n", ret);
    }
    return 0;
}

File: baekjoon/dynamic/6359/Main.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int testCase = sc.nextInt();
        for(int tc=0; tc<testCase; tc++) {
            int n = sc.nextInt();
            int ret = 1;
            for(int i=2; i<=n; i++) {
                if(i*i <= n) ret++;
            }
            System.out.println(ret);
        }
    }
}